portfolio/src/main/webapp/script.js

```diff
@@ -43,14 +43,59 @@ function deleteData() {
     });
 }
 
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyDjhk-U7NK6nQwmqSScCuZvO3l3_Q88Kps';
+script.defer = true;
+script.async = true;
+
+// Attach your callback function to the `window` object
+window.initMap = function() {
+  // JS API is loaded and available
+};
+
+// Append the 'script' element to 'head'
+document.head.appendChild(script);
+
 function createMap() {
+    document.head.appendChild(script);
```

Comment:
this seems like a repeated line. It also shouldn't be necessary if < script> is added  to the html file

Reply:
I deleted the repeat